travis/cfg/precommit/.prettierrc.yml

```diff
@@ -0,0 +1,8 @@
+# Prettier will complete this with settings from .editorconfig file.
+bracketSpacing: false
+printWidth: 119
+proseWrap: always
+semi: true
+trailingComma: "es5"
+xmlWhitespaceSensitivity: "strict"
+xmlSelfClosingSpace: false
```

Comment:
We may remove this one I think

Reply:
Too many changes are unavoidable anyway. It's expected the code to be re-written

Reply:
Yes, I know

We are deciding what checks we want and what we don't want

We even will disable more and more and custom the params I'm waiting for Nhomar review of the dummy

Reply:
Got it, thanks.